kubernetes/istio-15.yaml

```diff
@@ -83,7 +83,7 @@ data:
         params:
           quotas:
           - descriptor_name: RequestCount
-            max_amount: 5
+            max_amount: 5000
```

Comment:
Nope, this is a generated file. Somebody changed that value in mixer.yaml and did not run updateVersions.sh.